model/godleyTableWindow.cc

```diff
@@ -798,6 +798,55 @@ namespace minsky
       godleyIcon->deleteRow(r+1);
     requestRedraw();
   }
+  
+  string GodleyTableWindow::moveAssetClass(double x, double y)
+  {
+	x/=zoomFactor;
+	y/=zoomFactor;
+	unsigned c=colX(x);	
+	string tmpStr="";
+	if (clickType(x,y)==colWidget) {
+	    unsigned visibleCol=c-scrollColStart+1;
+        if (c<colWidgets.size() && visibleCol < colLeftMargin.size()) {
+		    auto& moveVar=godleyIcon->table.cell(0,c);
+		    auto oldAssetClass=classdesc::enumKey<GodleyAssetClass::AssetClass>(godleyIcon->table._assetClass(c));			
```

Comment:
I would leave the values as enums until the final construction of the string return value.

Reply:
Excellent suggestion! I will make that change today.